src/quota.rs

```diff
@@ -5,55 +5,142 @@ use std::time::Duration;
 
 /// A rate-limiting quota.
 ///
-/// Quotas are expressed in a positive number of "cells" (the number of positive decisions /
-/// allowed items) per unit of time.
+/// Quotas are expressed in a positive number of "cells" (the maximum number of positive decisions /
+/// allowed items until the rate limiter needs to replenish) and the amount of time for the rate
+/// limiter to replenish a single cell.
 ///
-/// Neither the number of cells nor the unit of time may be zero.
+/// Neither the number of cells nor the replenishment unit of time may be zero.
 ///
 /// # Burst sizes
 /// There are multiple ways of expressing the same quota: a quota given as `Quota::per_second(1)`
 /// allows, on average, the same number of cells through as a quota given as `Quota::per_minute(60)`.
 /// However, the quota of `Quota::per_minute(60)` has a burst size of 60 cells, meaning it is possible
 /// to accommodate 60 cells in one go, followed by a minute of waiting.
+///
+/// Burst size gets really important when you construct a rate limiter that should allow multiple
+/// elements through at one time (using [`RadeLimiter.check_n`](struct.RateLimiter.html#method.check_n)
```

Comment:
Should be `RateLimiter.check_n` here.

Reply:
Good catch! Guess that's why it wouldn't linkify automatically (: